src/services/replyService.js

```diff
@@ -0,0 +1,85 @@
+const db         = require('../models/sequelize');
+const models     = db.sequelize.models;
+const PostModel  = models.post;
+const UserModel  = models.user;
+const ReplyModel = models.reply;
+const Op         = db.Sequelize.Op;
+const _          = require('lodash');
+const client     = require('./steem');
+
+const {
+  GET_CONTENT_REPLIES,
+}                = require('../utils/constants');
+
+class ReplyService {
+  constructor(postFromOurDb) {
+    this.post = postFromOurDb;
+    this.postAuthor;
+    this.parent;
+  }
+
```

Comment:
Services shouldn't hold a state. It's just a lightweight module that can be injected anywhere. We should pass the post in as a parameter into the service methods ie:

```
class ReplyService {
   getAllComments(post) {
      if (this._isInvalidPost(post)) { throw new Error(`Expected post instance. Received: ${post}`); }
      return UserModel.findById(post.userId)
         .then(author => {
            // ...
         })
      })
      .catch(() => {});
}
```

Reply:
Ok cool - I had actually thought it was wierd after I wrote it.  Module can better. Ill add some edits to pass around post/author records instead of a class variables.

Reply:
Will address in the fetch-replies-replies pull request.

Reply:
👍